src/lib/openapi/spec/context-field-schema.ts

```diff
@@ -5,41 +5,59 @@ export const contextFieldSchema = {
     $id: '#/components/schemas/contextFieldSchema',
     type: 'object',
     additionalProperties: false,
+    description:
+        'A [context field](https://docs.getunleash.io/reference/unleash-context) used to configure strategies with data used to evaluate isEnabled calls',
     required: ['name'],
     properties: {
         name: {
+            description: 'The name of the context field',
             type: 'string',
+            example: 'userId',
         },
         description: {
+            description: 'The description of the context field.',
             type: 'string',
             nullable: true,
+            example: 'Used to uniquely identify users',
         },
         stickiness: {
+            description:
+                'Does this context field support being used for [stickiness](https://docs.getunleash.io/reference/stickiness) calculations',
             type: 'boolean',
+            example: true,
         },
         sortOrder: {
+            description:
+                'Used to have deterministic sorting of a list of context fields. Can also be used as a draw-breaker if list of context fields is sorted alphabetically.',
             type: 'number',
+            example: 900,
```

Comment:
Is this a `number` or an `integer`? Both are valid types in OpenAPI and if we always assume integers, then that may be a good way to make it more precise.